build.sbt

```diff
@@ -21,6 +21,14 @@ resolvers += Resolver.sonatypeRepo("snapshots")
 
 libraryDependencies += guice
 libraryDependencies += "org.scalatestplus.play" %% "scalatestplus-play" % "3.1.2" % Test
+libraryDependencies += "org.parboiled" %% "parboiled" % "2.1.4"
+libraryDependencies += "com.typesafe.akka" %% "akka-http" % "10.0.11"
```

Comment:
Please use the more current version 10.1.5. (see delphi-cli for migration)
Also, you might be able to just use the smaller akka-http-core library.